### DISCUSSION TITLE
How do I customize a plug-in？

### DISCUSSION THREAD

**Discussion by CembZy** [dbeaver/dbeaver#9324] [DBeaver Development]
I developed a Hello Word plug-in on Eclipse, exported it as a JAR, and then copied it into the Dbeaver plugins directory. However, after restarting dBeaver, my custom plug-in doesn't appear on the screen. How can I import my plug-in into DBeaver?

**Comment by c0nsy** [comment]
@CembZy  Any update on this? 
Thanks

**Reply by TB-HunBoka** [reply]
> @CembZy Any update on this? Thanks

Are you looking for this?
https://dbeaver.com/docs/dbeaver/Optional-extensions/
You can add local repository to the repository manager.
